### DISCUSSION TITLE
time zone displacement out of range: "$NOW(-30 days)"

### DISCUSSION THREAD

**Discussion by sambitraze** [directus/directus#17025] [Archive]
const DownloadData2 = await DownloadDataCollection.readByQuery({
			filter: {
				status: {
					_eq: "available",
				},
				date_updated: {
					_lt: "$NOW(-30 days)",
				},
			}
		});

**Comment by br41nslug** [comment]
What is the question?

**Reply by sambitraze** [reply]
I am using the above filter inside a schedule cron extension but after I use the date_upated filter with _lt  = "$NOW(-30 days)" it's giving me an error if I just pass the filter as parameter on the URL  its works fine without any error

**Comment by tyler71** [comment]
Stumbled upon this error, this is how I solved it:
```javascript
const today = new Date();
const thirtyDaysAgo = new Date(today);
thirtyDaysAgo.setDate(today.getDate() - 30);

date_updated: {
    _lt: thirtyDaysAgo.toISOString(),
},
```
